lib/constable_web/templates/announcement/show.html.eex

```diff
@@ -10,15 +10,17 @@
       <%= if @subscription do %>
         <%= link to: announcement_subscription_path(@conn, :delete, @announcement.id),
           method: :delete,
-          class: "button button-unsubscribe",
+          class: "unsubscribe-to",
           data: [turbolinks: "refresh"] do %>
-          <%= gettext("Unsubscribe") %>
+          <img class="subscribe-icons" src="/images/icon-check-white.svg" />
```

Comment:
May be good to set an empty `alt` for this and the other image so it's not read by screen readers. Alternatively, since it's just decoration, could be a `:before` element with empty content and bg image that calls that svg.

Reply:
Great points. I like the pseudo element idea. Let me give it a go.